relayer/strategies.go

```diff
@@ -22,8 +22,9 @@ type ActiveChannel struct {
 }
 
 const (
-	ProcessorEvents string = "events"
-	ProcessorLegacy        = "legacy"
+	ProcessorEvents              string = "events"
+	ProcessorLegacy                     = "legacy"
+	DefaultClientUpdateThreshold        = 6 * time.Hour
```

Comment:
In combination with this suggestion https://github.com/cosmos/relayer/pull/1008/files#r1000932480:

```suggestion
	DefaultClientUpdateThreshold        = 0 * time.Millisecond
```